### PR TITLE
Update Dragen WTS input options

### PR DESCRIPTION
**Pull request by skanwal**
Adding options for read trimming

### CODE REVIEW

File: tools/dragen-transcriptome/4.2.4/dragen-transcriptome__4.2.4.cwl [hunk 1]
```diff
@@ -239,6 +239,89 @@ inputs:
...
       separate: False
+  read_trimmers:
+    label: read trimming
+    type: string?
+    doc: |
+      To enable trimming filters in hard-trimming mode, set to a comma-separated list of the trimmer tools 
+      you would like to use. To disable trimming, set to none. During mapping, artifacts are removed from all reads.
+      Read trimming is disabled by default.
+    inputBinding:
+      prefix: "--read-trimmers"
```
Comment (alexiswl) [inline tools/dragen-transcriptome/4.2.4/dragen-transcriptome__4.2.4.cwl:250]: inputBinding should be of the format:

```
prefix: "--parameter-key="
separate: False
```

File: tools/dragen-transcriptome/3.9.3/dragen-transcriptome__3.9.3.cwl [hunk 2]
```diff
@@ -353,6 +353,61 @@ inputs:
...
       prefix: "--rrna-filter-contig"
+  read_trimmers:
+    label: read trimming
+    type: string?
+    doc: |
+      To enable trimming filters in hard-trimming mode, set to a comma-separated list of the trimmer tools 
+      you would like to use. To disable trimming, set to none. During mapping, artifacts are removed from all reads.
+      Read trimming is disabled by default.
+    inputBinding:
+      prefix: "--read-trimmers"
```
Comment (alexiswl) [inline tools/dragen-transcriptome/3.9.3/dragen-transcriptome__3.9.3.cwl:364]: inputBinding should be of the format:

```
prefix: "--parameter-key="
separate: False
```

**Review by alexiswl — CHANGES_REQUESTED** [review]

File: tools/dragen-transcriptome/4.2.4/dragen-transcriptome__4.2.4.cwl (hunk 1 above)
Comment (skanwal) [reply tools/dragen-transcriptome/4.2.4/dragen-transcriptome__4.2.4.cwl]: Updated in https://github.com/umccr/cwl-ica/pull/391/commits/8c69f8334868ef36364852786fc37e3e1f8cdb22

File: tools/dragen-transcriptome/3.9.3/dragen-transcriptome__3.9.3.cwl (hunk 2 above)
Comment (skanwal) [reply tools/dragen-transcriptome/3.9.3/dragen-transcriptome__3.9.3.cwl]: We were still using older inputBinding option for v3.9. I have updated all inputs to use consistent bindings.

Comment (skanwal) [reply tools/dragen-transcriptome/3.9.3/dragen-transcriptome__3.9.3.cwl]: https://github.com/umccr/cwl-ica/pull/391/commits/8c69f8334868ef36364852786fc37e3e1f8cdb22

**Review by alexiswl — APPROVED** [review]

**Pull request merged**